sections/05_treat.md

```diff
@@ -375,15 +375,9 @@ atom-atom interactions and the forces that govern binding affinity.
 
 #### *De novo* drug design
 
-Whereas the goal of virtual screening is to find active molecules by predicting
```

Comment:
Moved the citation to the subsequent paragraph in michaelmhoffman/deep-review@904b89b9c847b90454e23ae690b99273ad749957